compiler/noirc_driver/src/lib.rs

```diff
@@ -193,7 +193,10 @@ pub fn check_crate(
     let macros: Vec<&dyn MacroProcessor> = if disable_macros {
         vec![]
     } else {
-        vec![&aztec_macros::AztecMacro as &dyn MacroProcessor]
+        vec![
+            &aztec_macros::AztecMacro as &dyn MacroProcessor,
+            &noirc_macros::AssertMessageMacro as &dyn MacroProcessor,
```

Comment:
This is going to break vanilla noir programs if we have dynamic assert messages and disable macros. Probably worth scoping the flag to just cover aztec macros.

Reply:
I put the assert message macro processor in the macros vec for when `disable_macros` is true.